test/system/event_log_interaction_test.rb

```diff
@@ -38,14 +37,42 @@ class EventLogInteractionTest < ApplicationSystemTestCase
       end
 
       assert_difference '@user2.call_list_entries.count', 1 do
-        within :css, '#activity_log_content' do
```

Comment:
Weird spurious failures here for a reason I couldn't figure out.

Reply:
isnt this redundant to the test on line 35 above anyway?